examples/storm-pmml-examples/src/main/resources/KNIME_PMML_4.1_Examples_single_audit_logreg.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@ptgoetz I found [PMML	Notice and License](http://dmg.org/documents/dmg-pmml-license-2016.pdf).